projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/JobsService.java

```diff
@@ -240,9 +240,8 @@ public void updateLastExecution(
          return;
       }
 
-      dataJob.setLastExecutionStatus(dataJobExecution.getStatus());
-      dataJob.setLastExecutionEndTime(dataJobExecution.getEndTime());
-      dataJob.setLastExecutionDuration((int)(dataJobExecution.getEndTime().toEpochSecond() - dataJobExecution.getStartTime().toEpochSecond()));
+      log.debug("Updating last execution of data job {}: {}", dataJob.getName(), dataJobExecution);
```

Comment:
Does the data job execution have a string representation, or would it be logged as something like `<DataJobExecution object at some-address>`?

Reply:
It has automatic `toString` implementation via lombok: https://github.com/vmware/versatile-data-kit/blob/main/projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/model/DataJobExecution.java#L20

Reply:
Thanks